mad-dog/src/helpers.js

```diff
@@ -55,6 +56,57 @@ const addAndUpgradeUsers = async (
   const existingUserIds = []
   const walletIndexToUserIdMap = {}
 
+  await _addUsers({ userCount, executeAll, executeOne, existingUserIds, addedUserIds, walletIndexToUserIdMap })
+  await _upgradeToCreator(executeAll, executeOne, numCreatorNodes)
+
+  // Map out walletId index => userId
+  return walletIndexToUserIdMap
+}
+
+const addUsers = async (
+  userCount,
+  numCreatorNodes,
```

Comment:
this field is never used here or below on line 84

Reply:
good catch, removing!